src/content/docs/ai-monitoring/configure-ai-monitoring.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Configure AI monitoring'
+metaDescription: 'You can apply certain configurations to your APM agents to change how your AI data appears in New Relic.'
+freshnessValidatedDate: never
+---
+
+Once you install AI monitoring, you can configure the default behavior of the agent or update your app to collect different kinds of data. 
+
+## Configure the agent [#configure-agents]
+
+Update default agent behavior for AI monitoring at these agent configuration docs: 
```

Comment:
I'd drop a note that a lot of these config options must be decided on during set-up and cannot be changed later (that's my understanding)